src/control-plane/backend/lambda/api/store/aws/quicksight.ts

```diff
@@ -140,21 +140,34 @@ export const registerQuickSightUser = async (email: string, username?: string) =
   return registerQuickSightUserByRegion(identityRegion, email, username);
 };
 
+export const registerClickstreamUser = async () => {
+  const identityRegion = await getIdentityRegion();
+  await registerEmbeddingUserByRegion(identityRegion);
+  await registerQuickSightUserByRegion(identityRegion, QUICKSIGHT_EMBED_NO_REPLY_EMAIL, QUICKSIGHT_DASHBOARD_USER_NAME);
```

Comment:
Should we have a user as the owner of explorer dashboards of the QuickSight **per region**?